### PR TITLE
Add `related` field to web UI.

### PR DESCRIPTION
**Pull request by hogo6002**
Fix issue https://github.com/google/osv.dev/issues/1734
image

### CODE REVIEW

**Review by michaelkedar — COMMENTED** [review]
Looks good!
We probably also want to display the vulns that list the current vuln as their related also.
So in your example, CVE-2020-23804's page should also show DLA-3620-1 as related.

**Comment by hogo6002** [comment]
> DLA-3620-1

I am thinking this computation should be done by PR https://github.com/google/osv.dev/pull/1733 right? Once that PR merged, CVE-2020-23804's related field will have DLA-3620-1 and then show it here.

**Review by oliverchang — APPROVED** [review]

File: gcp/appengine/frontend_handlers.py [hunk 1]
```diff
@@ -579,3 +579,9 @@ def package_in_ecosystem(package):
   if ecosystem in osv.ecosystems.package_urls:
     return osv.ecosystems.package_urls[ecosystem] + package['name']
   return ''
+
+
+@blueprint.app_template_filter('osv_has_bug')
```
Comment (cuixq) [inline gcp/appengine/frontend_handlers.py:584]: rename it to `osv_has_vuln`?

**Review by cuixq — APPROVED** [review]

File: gcp/appengine/frontend_handlers.py (hunk 1 above)
Comment (hogo6002) [reply gcp/appengine/frontend_handlers.py]: Thanks!

**Pull request merged**